resources/assets/components/InboxItem/index.js

```diff
@@ -22,8 +22,10 @@ class InboxItem extends React.Component {
     const post_id = post['id'];
     const signup = this.props.details.signup;
 
+    var posts = signup.posts ? signup.posts : post.siblings.data;
```

Comment:
@DFurnes would it be better to pass props like this: 
`key={key} post={post} campaign={campaign} signup={post.signup.data}` instead of `details={{post: post, campaign: campaign, signup: this.state.signups[post.signup_id]}}` [here](https://github.com/DoSomething/rogue/blob/master/resources/assets/components/CampaignSingle/index.js#L109)?